boards/metro_m0/src/lib.rs

```diff
@@ -254,3 +252,19 @@ pub fn usb_bus(
         usb,
     ))
 }
+
+#[cfg(feature = "usb")]
+#[deprecated(
+    since = "0.8.0",
```

Comment:
I had written this awhile back. I don't think the BSPs have been released yet, but if `0.9.0` gets published then I can update this.